ci/setup_env.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash -e
+
+
+# edit the locale file if needed
+function edit_init()
+{
+    if [ -n "$LOCALE_OVERRIDE" ]; then
+        echo "[Adding locale to the first line of pandas/__init__.py]"
+        rm -f pandas/__init__.pyc
+        sedc="3iimport locale\nlocale.setlocale(locale.LC_ALL, '$LOCALE_OVERRIDE')\n"
+        sed -i "$sedc" pandas/__init__.py
+        echo "[head -4 pandas/__init__.py]"
+        head -4 pandas/__init__.py
+        echo
+        sudo locale-gen "$LOCALE_OVERRIDE"
+    fi
+}
+
+edit_init
+
+HOME_DIR=$(pwd)
+echo
+echo "HOME_DIR: $HOME_DIR"
```

Comment:
`HOME_DIR` is not used, I would remove these 3 lines, I don't think the echo is really useful.